react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -133,6 +191,32 @@ class ToolbarButton extends Component {
     }
 
     /**
+     * Parses the props and state to find any popup that should be displayed
+     * and returns an object desribing how the popup should display.
+     *
+     * @private
+     * @returns {Object|null}
+     */
+    _getPopupDisplayConfiguration() {
+        const { button, tooltipPosition } = this.props;
+        const { popups, popupDisplay } = button;
```

Comment:
The microphone button supports three popups. https://github.com/jitsi/jitsi-meet/blob/c42f1704ff6ebb23c3f5e6f0846bb453c0481b31/react/features/toolbox/defaultToolbarButtons.js#L288